.github/workflows/build-package-docs.yaml

```diff
@@ -120,3 +131,88 @@ jobs:
         name: package-docs-build
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
+
+  deploy-package-docs:
+    if: fromJSON(github.event.inputs.deploy)
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    environment:
+      name: deploy-package-docs
+      url: https://github.com/ansible/ansible-documentation
+    env:
+      TARGET: ${{ github.event.inputs.deployment-environment }}
+      DEST_REPO: ansible-community/package-doc-builds
+      USER_EMAIL: "41898282+github-actions[bot]@users.noreply.github.com"
+      USER_NAME: "github-actions[bot]"
+    steps:
+    - name: Download the build archive
+      uses: actions/download-artifact@v4
+      with:
+        name: package-docs-build
+
+    - name: Extract the tarball
+      run: |
+        set -xeEuo pipefail
+        mkdir -pv extracted-docs
```

Comment:
I'm 99% sure extraction creates this directory already. Have you tried?
```suggestion
```

Reply:
I tried that one out already but tar doesn't create the directory. Consulting the man page and perhaps `--one-top-level` could be an option as it extracts into a dir with the base name of the archive. However that name is pretty long given the descriptive name.

Maybe if we could use a wildcard in the source directory for the rsync step... I'll try it.

Reply:
Yes, the `--one-top-level` flag seems to work https://github.com/ansible/ansible-documentation/pull/1353/commits/79b9b45e02cee3b7a8749f48973ef7da7780d4a6